app/views/users/_profile_cumulativestats.html.haml

```diff
@@ -0,0 +1,46 @@
+%h3
+  = t("users.contribution_statistics")
+%br/
+%h5
+  = t("users.user_impact")
+  #{@user.username}
```

Comment:
Instead of handling this separately, you should pass the username to the message function. Doing it this way is called a 'lego' message (because it builds the whole from parts), and it's bad for internationalization because not all languages will put the parts in the same order.